blog/_posts/2018-10-04-scaladoc-tables.md

```diff
@@ -97,7 +97,7 @@ For a smorgasbord of examples take a look at [examples](https://janekdb.github.i
 
 ### Dotty and Scala 3
 
-With `dotty` now boosting its share of the development horizon it's important to place GFM into a [dottydoc](https://dotty.epfl.ch/docs/usage/dottydoc.html) and hence Scaladoc 3 context.
+With `dotty` now boosting its share of the development horizon it's important to place GFM into a [dottydoc](https://docs.scala-lang.org/scala3/guides/scaladoc/) and hence Scaladoc 3 context.
```

Comment:
The page dottydoc says its content is outdated: https://github.com/lampepfl/dotty/blob/fa08d004308ff4dee799aafa30939e0647018d72/docs/_docs/usage/dottydoc.md